koi/koi.py

```diff
@@ -106,8 +106,8 @@ def _get_extra_args(self):
             "variations": str(self.variations.value()),
             "prompt": self._prompt_text(),
             "steps": str(self.steps.value()),
-            "seed": str(self.base_seed.value()),
-            "sketch_strength": str(1.0-self.sketch_strengh.value()),
```

Comment:
the reason it is `1.0-sketch_strength` is becuase higher values of strength actually leads to the image having less of an influence.

In the original repo the parameter is called `strength` and refers to how much noise gets added to the `init_image`

Flipping the values by subtracting 1.0 leads to a more intuitive param for end users imo.

https://github.com/CompVis/stable-diffusion/blob/69ae4b35e0a0f6ee1af8bb9a5d0016ccb27e36dc/scripts/img2img.py#L159-L163